open_buildings/settings.py

```diff
@@ -0,0 +1,43 @@
+from enum import Enum
+from typing import Dict
+from pydantic import BaseModel
+
+class Source(Enum):
+    GOOGLE = 1
+    OVERTURE = 2
+
+class Format(Enum):
+    SHAPEFILE = 1
+    GEOJSON = 2
+    GEOPACKAGE = 3
+    FLATGEOBUF = 4
+    PARQUET = 5
+
+
+class SourceSettings(BaseModel):
+    base_url: str
+    hive_partitioning: bool
+
+class SettingsSchema(BaseModel):
+    sources: Dict[Source, SourceSettings]
+    extensions: Dict[Format, str]
+
+settings = SettingsSchema(
```

Comment:
Love this, makes a ton of sense. I'm learning more python just reading this PR :)